qemu/tests/cfg/virtio_fs_share_data_multi_backend.cfg

```diff
@@ -0,0 +1,157 @@
+- virtio_fs_share_data_multi_backend:
+    no RHEL.6 RHEL.7 RHEL.8.0 RHEL.8.1
+    no Win2008 Win7
+    no Host_RHEL.m6 Host_RHEL.m7 Host_RHEL.m8.u0 Host_RHEL.m8.u1
+    type = virtio_fs_share_data
+    virt_test_type = qemu
+    required_qemu = [4.2.0,)
+    s390, s390x:
```

Comment:
Since patch: `https://github.com/autotest/tp-qemu/pull/2982` has been merged, please `git fetch --all` patch and help to update the S390 configuration part synchronously, thanks a lot.